test/fixture-utils.js

```diff
@@ -5,19 +5,7 @@ const fs = require('fs').promises;
 const snowpack = require('snowpack');
 const assert = require('assert');
 
-const UTF8_FRIENDLY_EXTS = [
-  'css',
-  'html',
-  'js',
-  'map',
-  'jsx',
-  'ts',
-  'tsx',
-  'svelte',
-  'svg',
-  'vue',
-  'json',
-]; // only read non-binary files (add more exts here as needed)
+const IGNORE_EXTS = ['.png', '.jpg', '.ico']; // only read non-binary files (add more exts here as needed)
```

Comment:
There was no way to use the old pattern to match dotfiles. On the bright side, I actually got a lot of false negatives while working on tests in the old system, where I thought a file wasn't built when really its extension just wasnt in this array.

We have way fewer non-binary files in tests than we think and the test runner should break loudly if a new one is ever added, so just ignore them here instead.